mapper/thin_edge_json/src/serialize.rs

```diff
@@ -21,6 +21,9 @@ pub enum ThinEdgeJsonSerializationError {
 
     #[error("Serializer produced invalid Utf8 string")]
     InvalidUtf8ConversionToString(std::string::FromUtf8Error),
+
+    #[error("transparent")]
+    JsonWriterError(#[from] JsonWriterError),
```

Comment:
I do think this error case can be removed. See my comment on `JsonWriterError`.

Reply:
Or we keep it (json writing might fail in some cases... maybe in the future) and simply make the JsonWriterError for now an empty enum. I am happy with either way.